acceptance_tests/features/steps/set_ready_for_live.py

```diff
@@ -35,11 +36,7 @@ def prepare_collection_exercises(_, survey, period):
         # form type hard coded to 0001 for all ces to simplify testing
         collection_instrument_controller.upload_seft_collection_instrument(ce['id'], ci_path, '0001')
 
-    for i in range(5):
-        state = collection_exercise_controller.get_collection_exercise(s_id, period)['state']
-        if state == 'READY_FOR_REVIEW':
-            break
-        time.sleep(1)
+    browser.is_text_present('READY_FOR_REVIEW', wait_time=2)
```

Comment:
Not sure if this will work because the page won't be updated asynchronously although the data is